### DISCUSSION TITLE
父母看到孩子變老的反應是？

### DISCUSSION THREAD

**Discussion by Sherry9106** [1101-GS4719-C/Python-Community-GS4719-C#1617] [General]
最近Dcard發布了一個影片，叫做【80歲的我們】特別企劃，主題是「我能看著爸媽變老，但他們卻看不到我老的樣子」，裡面有一句話是說，大部分的父母應該都看不到自己的小孩變老吧 ? 看到的時候其實有點難過，因為我媽曾對我說過 : 我沒辦法陪你到老。
每次要從家裡回學校的時候，我爸媽就會狂問我需不需要帶東西回去，然後塞到我都快拿不動了才停，然後我吃到圓滾滾的時候他們又會開玩笑，如果他們看到我突然變超老的樣子，應該會先狂笑吧 ? 然後笑完再說我怎麼總是搞一些亂七八糟的東西哈哈哈，想問大家的父母如果看到變老的自己會有什麼反應呢 ? 

https://www.youtube.com/watch?v=VkxLj1HlIao&t=56s

**Comment by v0ouo** [comment]
我個人認為，父母看我們變老不一定要看我們到一定的歲數，拿我的父母為例，它們常常一直說因為每天都能看到我沒什麼成長的感覺，但是又常常說幾年前我還怎樣怎樣之類的。當我行為不再像以前一樣，而是漸趨成熟，變得不像小孩子，就是我父母真的認為我變老的時候了，他們應該會覺得很欣慰吧!       (補充一下我常常因為白頭髮太多被說像老人家)

**Comment by WeiYen1103** [comment]
我覺得那部影片到後面很感人，畢竟真的沒有甚麼機會可以讓父母看到我們老了的樣子，現在大學的我，也不能像小時候常常陪在家人身邊，總覺得他們老得好快，想趕快長大照顧他們，但如果有這個機會，我也會想讓我家人看看我老了之後長怎樣。

**Comment by THH1118** [comment]
爸媽跟長輩最看不準的就是我們的身材，每次回家明明就變胖了結果長輩看到我們第一句話都是「怎麼變瘦了」，不要再餵食我了啊啊啊

**Comment by uwje** [comment]
我認為大概會是擔心吧，擔心你為何突然變那麼老。

**Comment by yuku-huang** [comment]
如果我爸媽看到我突然變老，應該會一邊笑一邊假裝生氣說「你到底又在搞什麼奇怪的實驗」。但過一下又會沉默，看著我很久，然後說「你怎麼老得這麼快，我還沒看夠你現在的樣子」。其實他們可能很難接受這個畫面吧，因為在他們眼裡，小孩永遠都是小孩。他們會想參與我們的一生，但現實就是時間不允許，所以他們才會把所有關心都擠在我們出門前那幾分鐘，連零食都不忘幫我們塞包包裡。想一想，這種「看不到你老去」的遺憾，其實是愛的其中一種表現。
